staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -332,18 +332,23 @@ func (s *EtcdOptions) maybeApplyResourceTransformers(c *server.Config) (err erro
 
 		c.ResourceTransformers = dynamicTransformers
 		if !s.SkipHealthEndpoints {
-			c.AddHealthChecks(dynamicTransformers)
+			s.addHealthChecks(c, dynamicTransformers)
 		}
 	} else {
 		c.ResourceTransformers = encryptionconfig.StaticTransformers(encryptionConfiguration.Transformers)
 		if !s.SkipHealthEndpoints {
-			c.AddHealthChecks(encryptionConfiguration.HealthChecks...)
+			s.addHealthChecks(c, encryptionConfiguration.HealthChecks...)
 		}
 	}
 
 	return nil
 }
 
+func (s *EtcdOptions) addHealthChecks(c *server.Config, healthChecks ...healthz.HealthChecker) {
```

Comment:
`s` is not used and this need to indicate that `livez` is not wired.

```suggestion
func addHealthChecksWithoutLivez(c *server.Config, healthChecks ...healthz.HealthChecker) {
```